server.go

```diff
@@ -2021,7 +2021,8 @@ func (s *server) createNewHiddenService() error {
 	onionCfg := tor.AddOnionConfig{
 		VirtualPort: defaultPeerPort,
 		TargetPorts: listenPorts,
-		Store:       tor.NewOnionFile(s.cfg.Tor.PrivateKeyPath, 0600),
+		Store: tor.NewOnionFile(s.cfg.Tor.PrivateKeyPath, 0600,
+			s.cfg.Tor.EncryptKey, s.cc.keyRing),
```

Comment:
Nit: function calls with arguments that span multiple lines should follow the style

```
tor.NewOnionFile(
    privateKey, encryptKey, ...,
    keyRing, ...,
)
```